cni/network/network.go

```diff
@@ -145,12 +146,6 @@
 	common.LogNetworkInterfaces()
 
 	// Initialize network manager.
-	err = plugin.nm.Initialize(config, rehydrateNetworkInfoOnReboot)
```

Comment:
Why did we move this?

Reply:
we restore state inside this call which we have to do after lock